### PR TITLE
Prepare 7.0.0 release, upgrade to OTP24

### PR DESCRIPTION
**Pull request by radrow**
BEFORE MERGE
- [x] Upgrade lock after `aebytecode` 3.1.1 is released

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+### Changed
+### Removed
+
+## [7.0.0]
+### Added
 - Added support for `EXIT` opcode via `exit : (string) => 'a` function (behaves same as `ABORT`, but consumes all gas).
 - Compiler warnings for the follwing: shadowing, negative spends, division by zero, unused functions, unused includes, unused stateful annotations, unused variables, unused parameters, unused user-defined type, dead return value.
```
Comment (hanssv) [inline CHANGELOG.md:15]:
```suggestion
- Compiler warnings for the following: shadowing, negative spends, division by zero, unused functions, unused includes, unused stateful annotations, unused variables, unused parameters, unused user-defined type, dead return value.
```

**Review by hanssv — DISMISSED** [review]

**Review by dincho — DISMISSED** [review]
CI changes seems harmless

File: .circleci/config.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   aebuilder:
     docker:
-      - image: aeternity/builder:xenial-otp21
+      - image: aeternity/builder:bionic-otp24
```
Comment (ghallak) [inline .circleci/config.yml:6]: @radrow You said this is going to work with otp25?

**Review by hanssv — DISMISSED** [review]

**Review by ghallak — APPROVED** [review]
rebar.lock is updated after aebytecode 3.1.1 is released

**Review by hanssv — APPROVED** [review]

**Pull request merged**